src/backend/d3d12/TextureD3D12.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright 2017 The NXT Authors
```

Comment:
The changes introduced here are already present in the "binding model" PR and look like an old version of it.